src/app/core/components/left-nav/left-nav.component.html

```diff
@@ -1,55 +1,59 @@
-<p-tabView
-  styleClass="dark tab-left-nav"
-  (onChange)="onSelectionChangedByIdx($event)"
-  [(activeIndex)]="activeTabIndex"
-  *ngIf="currentUser$|async as currentUser"
->
-
-  <p-tabPanel>
-    <ng-template pTemplate="header" tabindex="5">
-      <span class="indicator dark"></span>
-      <i class="fa fa-hand-paper"></i>
-      <span class="p-tabview-title" i18n="|Tab name">Activities</span>
-    </ng-template>
-  </p-tabPanel>
-
-  <p-tabPanel>
-    <ng-template pTemplate="header" tabindex="6">
-      <span class="indicator dark"></span>
-      <i class="fa fa-graduation-cap"></i>
-      <span class="p-tabview-title" i18n="|Tab name">Skills</span>
-    </ng-template>
-  </p-tabPanel>
-
-  <p-tabPanel *ngIf="activeTabIndex === 2 || !currentUser.tempUser">
-    <ng-template pTemplate="header" tabindex="7">
-      <span class="indicator dark"></span>
-      <i class="fa fa-users"></i>
-      <span class="p-tabview-title" i18n="|Tab name">Groups</span>
-    </ng-template>
-  </p-tabPanel>
-
-</p-tabView>
-
-<ng-container *ngIf="$any($any(dataSources[activeTabIndex]!.state$) | async) as state">
-
-  <div class="spinner" *ngIf="state.isFetching">
-    <alg-loading size="medium"></alg-loading>
-  </div>
-
-  <alg-error
-    *ngIf="state.isError"
-    class="light"
-    i18n-message message="Unable to load the list."
-    [showRefreshButton]="true"
-    refreshButtonType="retry"
-    (refresh)="retryError()"
-  ></alg-error>
-
-  <alg-left-nav-tree
-    *ngIf="state.isReady"
-    [data]="state.data"
-    [elementType]="$any(['activity', 'skill', 'group'][activeTabIndex])"
-  ></alg-left-nav-tree>
+<ng-container *ngIf="activeTab$ | async as activeTab;">
```

Comment:
I might have found a bug (not linked to this PR)

Steps to reproduce:

Case 1:
1. Navigate to [first skill page](https://dev.algorea.org/en/#/skills/by-id/3000;path=;attempId=0/details)
2. Click on `Breadth-First Search (BFS)`
3. Click on `Depth First Search (DFS)`

:arrow_right: In the left nav, we are redirected to the root

Case 2:
1. Navigate to [first skill page](https://dev.algorea.org/en/#/skills/by-id/3000;path=;attempId=0/details)
2. Click on `Breadth-First Search (BFS)`
3. Click on `SASA`
4. Click on `Depth First Search (DFS)`

:arrow_right: In the left nav, we are **not** redirected to the root.

Is that the expected behavior ?